code/src/components/AddTask.js

```diff
@@ -0,0 +1,80 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React, { useState } from 'react'
+import { useDispatch } from 'react-redux'
+import tasks from 'reducers/tasks'
+import styled from 'styled-components'
+
+const AddTaskForm = styled.form`
+  padding-top: 2.5em;
+  padding-bottom: 1.5em;
+  display: flex;
+  align-items: center;
+`
+
+const FormLabel = styled.p`
+  display: inline-block;
+  padding:0;
+`
```

Comment:
For better accessibility you might want to to change the `styled.p` to `styled.label`